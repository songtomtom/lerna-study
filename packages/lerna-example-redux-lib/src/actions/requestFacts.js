export const REQUEST_FACTS = 'REQUEST_FACTS'

export function requestFacts() {
  return {
    type: REQUEST_FACTS
  }
}
