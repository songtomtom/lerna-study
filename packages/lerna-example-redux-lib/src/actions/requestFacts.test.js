import { requestFacts } from './requestFacts'

test('returns correct value', () => {
  expect(requestFacts()).toMatchSnapshot();
})
