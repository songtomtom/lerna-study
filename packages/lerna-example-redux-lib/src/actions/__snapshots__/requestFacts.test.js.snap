// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns correct value 1`] = `
Object {
  "type": "REQUEST_FACTS",
}
`;
